src/register/register.service.ts

```diff
@@ -46,6 +46,15 @@ export class RegisterService {
     }
   }
 
+  async resendEmailToSetPassword(email: string) {
+    const user = await this.usersService.findOneByEmail(email);
+    await this.mailService.sendMail(
+      email,
+      'USTAW NOWE HASŁO NA PLATFORMIE HEADHUNTERS by MegaK',
```

Comment:
Nie wiem czy to istotne ale jak już robię review... :P
```suggestion
      'USTAW NOWE HASŁO NA PLATFORMIE HEADHUNTER by MegaK',
```